app/views/admin/calendar/_event.slim

```diff
@@ -0,0 +1,10 @@
+- if cal_event[:event_type] == "loan_start"
+  div.calendar-event.loan-starts
+    = render "admin/calendar/event_loan", cal_event: cal_event
+- elsif cal_event[:event_type] == "loan_end"
+  div.calendar-event.loan-end
+    = render "admin/calendar/event_loan", cal_event: cal_event
+- elsif cal_event[:event_type] == "project_step"
+  = render "admin/calendar/event_step", cal_event: cal_event
+- else
```

Comment:
Actually, the better way to do this is to use a `case` statement. Let me know if you need help with that.